packages/core/package.nls.json

```diff
@@ -300,6 +300,7 @@
     "AWS.amazonq.featureDev.pillText.sendFeedback": "Send feedback",
     "AWS.amazonq.featureDev.pillText.selectFiles": "Select files for context",
     "AWS.amazonq.featureDev.pillText.retry": "Retry",
+    "AWS.amazonq.featureDev.pillText.retryOrAddNewPrompt": "Retry or write a new prompt",
```

Comment:
is this copy approved to use?

Reply:
It will have a slight change for that. I'll push an updated string.